src/main/java/uk/ac/cam/cl/dtg/segue/api/EmailFacade.java

```diff
@@ -496,4 +498,109 @@ public final Response sendEmailsToUserIds(@Context final HttpServletRequest requ
 
         return Response.ok().build();
     }
+
+    /**
+     * sendemailwithuserids allows sending an email to a given list of userids.
+     *
+     * This method will return serialised html that displays an email object
+     *
+     * @param request
+     *            - so that we can allow only logged in users to view their own data.
+     * @param emailTypeString
+     *            - the type of e-mail that is being sent.
+     * @param emailTemplates
+     *            - map which must contain the plaintextTemplate and htmlTemplate
+     * @return Response object containing the serialized content object. (with no levels of recursion into the content)
+     */
+    @POST
+    @Path("/email/sendcontentemailwithuserids/{emailtype}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @GZIP
+    @ApiOperation(value = "Send an email to a list of user IDs.")
+    public final Response sendContentEmailsToUserIds(@Context final HttpServletRequest request,
+                                              @PathParam("emailtype") final String emailTypeString,
+                                                     final ContentEmailDTO emailTemplates) {
+        if (Strings.isNullOrEmpty(emailTemplates.getPlainTextContent()) || Strings.isNullOrEmpty(emailTemplates.getHtmlContent()) || Strings.isNullOrEmpty(emailTemplates.getSubject())) {
+            return SegueErrorResponse.getBadRequestResponse("Response must include plaintextTemplate, htmlTemplate and emailSubject");
+        }
+
+        final String plaintextTemplate = emailTemplates.getPlainTextContent();
+        final String htmlTemplate = emailTemplates.getHtmlContent();
+        final String emailSubject = emailTemplates.getSubject();
+        final String overrideFromAddress = emailTemplates.getOverrideFromAddress();
```

Comment:
I think this isn't the right property for MailGun; `getOverrideEnvelopeFrom` is the one that is used to do the routing, the override From address is just the From address users see in their inbox!